packages/react-core/src/components/Menu/Menu.tsx

```diff
@@ -40,9 +40,9 @@ export interface MenuProps extends Omit<React.HTMLAttributes<HTMLDivElement>, 'r
   /** @beta Array of menus that are drilled in */
   drilledInMenus?: string[];
   /** @beta Callback for drilling into a submenu */
-  onDrillIn?: (fromItemId: string, toItemId: string, itemId: string) => void;
+  onDrillIn?: (event: any, fromItemId: string, toItemId: string, itemId: string) => void;
```

Comment:
do we want to be more specific with the event types here?

Reply:
I wonder if the onClick referenced that has an `any` type can be improved upon so that all of these other event types aren't consequently `any`. Without looking too much into it, it seems like the type could be `React.KeyboardEvent | React.MouseEvent` instead.

Reply:
@jeffpuzzo that seems to work for the most part! Had to make a few type tweaks for some of the function types defined in examples to get everything to compile but it seems to build without issue. Should have a PR up shortly.

Reply:
@wise-king-sullyman can you open a follow up breaking change issue to update the type for the `onClick` event.

Reply:
@tlabaj I'm not sure I understand, do you want me to revert the changes I made in 84b6467 and make a breaking followup to make the changes in v5 instead?